Lib/fontbakery/fonts_public.proto

```diff
@@ -12,7 +12,7 @@ message FamilyProto {
   required string name = 1;
   required string designer = 2;
   required string license = 3;
-  required string category = 4;
+  repeated string category = 4;  // Only the LAST value is used by Google Fonts
```

Comment:
Why!?

Also, why does the protobuf keep multiple category values if only the last item is supposed to be used? This sounds pretty much non-sense.

Reply:
I don't know, but that's why I'm only checking the last one...

Reply:
Who can we ask about it to figure out? @nathan-williams perhaps?